service/prometheus/prometheus.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	ClusterAnnotation = "giantswarm.io/prometheus-config-controller/cluster"
+	ClusterAnnotation = "giantswarm.io/prometheus-cluster"
```

Comment:
I miss the picture here. What is it used for? Who has it applied? Who reads it? What does it mean? Why not `giantswarm.io/prometheus-config-controller`?

Reply:
This is the annotation we apply to services we want `prometheus-config-controller` to scrape. The value is the job name.

Reply:
I feel we could bikeshed the actual value for a while, so i'm just gonna go with this :D